### ISSUE TITLE
Missing license info

### ISSUE THREAD

**Issue by sonalkr132** [glittergallery/GlitterGallery#308]
As far as I can tell, our repo doesn't have any license of use. We had agreed that we will be using MIT license.
There are a few images which should be removed as we don't really know what license they are using.
* [naruto.png](https://github.com/glittergallery/GlitterGallery/blob/master/spec/factories/files/naruto.png)
* [usercover.jpg](https://github.com/glittergallery/GlitterGallery/blob/master/public/usercover.jpg)

**Comment by sarupbanskota** [comment]
I introduced the usercover.jpg; it's under Creative Commons Zero. No worries there.

As for the other file, it appears like you introduced it, so you can look for a suitable replacement if you don't find an alternative.

**Comment by sonalkr132** [comment]
> As for the other file, it appears like you introduced it

Not really. It was introduced in a269b6. It doesn't matter tho. I will find a replacement. I hope it's okay if I send a PR which adds MIT license to project as well. We had agreed on it in mail.

**Comment by sonalkr132** [comment]
Do we have copyright? http://opensource.org/licenses/MIT asks for copyright info.
